frontend/src/components/common/ProfileSidebar.vue

```diff
@@ -121,14 +136,25 @@ import { IconSize } from '@/enum/IconEnum';
     font-weight: 700;
     display: flex;
     border: none;
-    background-color: transparent;
-    color: #000;
+    background-color: transparent !important;
+    color: $text-secondary !important;
     cursor: pointer;
 }
 
 .sign-out:hover {
     background-color: transparent;
     background-color: #ffffff;
+    color-scheme: $text-secondary !important;
```

Comment:
I am not sure why this is "color-scheme", did you mean to use "color"?
Without this added, does that work?

Reply:
it works the same but i'll use color to be more clear.